tegel/src/components/block/block.scss

```
@@ -4,6 +4,7 @@
   border-radius: 4px;
   padding: 16px;
   background-color: var(--sdds-block-bg-default);
+  font: var(--sdds-body-01);
```

Please include the font spacing variable too after this one.
```suggestion
  font: var(--sdds-body-01);
  letter-spacing: var(--sdds-body-01-ls);
```